requirements/constraints.txt

```diff
@@ -1,7 +1,5 @@
 jsonfield2<3.1.0
-django-waffle<2.1
 futures; python_version < "3"
 inflect<4.0.0
-edx-celeryutils==0.5.2 # This version is used with Koa.
-celery<5.0
-oauthlib==3.0.1
+edx-celeryutils==1.2.1 # This version is used with Maple.
+#oauthlib==3.0.1
```

Comment:
@BbrSofiane Is it intentionally commented out? I think we should keep or remove, but commenting this out will make confusions.

What is the reason this lib is not needed anymore?

Reply:
Yep, I should have added a note. 

I removed it because it was causing a conflict in the versions. I will remove it from the file.

```
There are incompatible versions in the resolved dependencies:
  oauthlib==3.0.1 (from -c requirements/constraints.txt (line 5))
  oauthlib==3.0.1 (from -c requirements/constraints.txt (line 5))
  oauthlib==3.0.1 (from -c requirements/constraints.txt (line 5))
  oauthlib==3.0.1 (from -c requirements/constraints.txt (line 5))
  oauthlib>=3.1.0 (from django-oauth-toolkit==1.3.3->-r requirements/base.in (line 8))
```